src/goto-symex/symex_assign.cpp

```diff
@@ -475,15 +475,11 @@ void goto_symext::symex_assign_symbol(
               << messaget::eom;
     });
 
-  const exprt assignment_guard =
-    make_and(state.guard.as_expr(), conjunction(guard));
-
-  const exprt original_lhs = get_original_name(l2_full_lhs);
   target.assignment(
```

Comment:
Random place: typo in the commit message: "this values" -> "these values", and also it should be "Inline variables" (plural)